test/monkey_test.rb

```diff
@@ -4,58 +4,69 @@
+    assert_equal({ elasticsearch: false, http: false, redis: false, grape: false, active_record: false }, Datadog::Monkey.get_patched_modules())
 
     Datadog::Monkey.patch_module(:redis)
     assert_equal(false, Datadog::Contrib::Elasticsearch::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::HTTP::Patcher.patched?)
     assert_equal(true, Datadog::Contrib::Redis::Patcher.patched?)
+    assert_equal(false, Datadog::Contrib::Grape::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::ActiveRecord::Patcher.patched?)
-    assert_equal({ elasticsearch: false, http: false, redis: true, active_record: false }, Datadog::Monkey.get_patched_modules())
+    assert_equal({ elasticsearch: false, http: false, redis: true, grape: false, active_record: false }, Datadog::Monkey.get_patched_modules())
 
     # now do it again to check it's idempotent
     Datadog::Monkey.patch_module(:redis)
     assert_equal(false, Datadog::Contrib::Elasticsearch::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::HTTP::Patcher.patched?)
     assert_equal(true, Datadog::Contrib::Redis::Patcher.patched?)
+    assert_equal(false, Datadog::Contrib::Grape::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::ActiveRecord::Patcher.patched?)
-    assert_equal({ elasticsearch: false, http: false, redis: true, active_record: false }, Datadog::Monkey.get_patched_modules())
+    assert_equal({ elasticsearch: false, http: false, redis: true, grape: false, active_record: false }, Datadog::Monkey.get_patched_modules())
 
     Datadog::Monkey.patch(elasticsearch: true, redis: true)
     assert_equal(true, Datadog::Contrib::Elasticsearch::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::HTTP::Patcher.patched?)
     assert_equal(true, Datadog::Contrib::Redis::Patcher.patched?)
+    assert_equal(false, Datadog::Contrib::Grape::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::ActiveRecord::Patcher.patched?)
-    assert_equal({ elasticsearch: true, http: false, redis: true, active_record: false }, Datadog::Monkey.get_patched_modules())
+    assert_equal({ elasticsearch: true, http: false, redis: true, grape: false, active_record: false }, Datadog::Monkey.get_patched_modules())
 
     # verify that active_record is not auto patched by default
     Datadog::Monkey.patch_all()
     assert_equal(true, Datadog::Contrib::Elasticsearch::Patcher.patched?)
     assert_equal(true, Datadog::Contrib::HTTP::Patcher.patched?)
     assert_equal(true, Datadog::Contrib::Redis::Patcher.patched?)
+    assert_equal(true, Datadog::Contrib::Grape::Patcher.patched?)
     assert_equal(false, Datadog::Contrib::ActiveRecord::Patcher.patched?)
-    assert_equal({ elasticsearch: true, http: true, redis: true, active_record: false }, Datadog::Monkey.get_patched_modules())
+    assert_equal({ elasticsearch: true, http: true, redis: true, grape: true, active_record: false }, Datadog::Monkey.get_patched_modules())
```

Comment:
I think we don't run the tests for Grape with old Ruby versions, but I would assume this global monkey patching test is run with oldies such as 1.9. So I'm wondering if we should keep this and just drop it, probably removing the `require 'grape'` in this test would be enough. Let's wait for the CI advice, what I'm saying is: for a first release, if Grape does not work with 1.9, let's save this for later and choose whatever path makes it straight to the release, we don't want to delay 0.7.0 for such a corner case. If all tests pass -> forget this remark.